kong/plugins/datadog/handler.lua

```diff
@@ -53,6 +53,11 @@ local function log(premature, conf, message)
   if premature then
     return
   end
+
+  if not message.api then -- unmatched apis are nil
```

Comment:
minor style nitpick, can we place the comment on the previous line instead of adjacent to the code here? thanks :)